sphinx/templates/quickstart/conf.py_t

```diff
@@ -132,6 +132,11 @@ latex_elements = {
     # Latex figure (float) alignment
     #
     # 'figure_align': 'htbp',
+
+    # Chinese language default package ctex
```

Comment:
because the empty dictionary by default overrides `default_latex_elements()`.
Another option would be comment whole `latex_elements={ }` in conf.py_t